docs/_media/non-delayed.md

```diff
@@ -0,0 +1,7 @@
+- This is from the `no-delay.md`
+
+```bash
+sleep 0 
+echo non-delayed
+exit 0
+```
```

Comment:
same as above ?

Reply:
Yep, mandatory part is `- This is ...` the bash is just filler for diff mismatch boost...